### PR TITLE
:arrow_up: updating base image to 1-104

### PR DESCRIPTION
**Pull request by goern**
Signed-off-by: Christoph Görn <goern@redhat.com>

### CODE REVIEW

**Comment by VannTen** [comment]
is this automated ? That should probably be redone on top of master

**Comment by goern** [comment]
> is this automated ? That should probably be redone on top of master

it's not automated, and I am just rebasing...

**Review by VannTen — COMMENTED** [review]
I'm not particularly familiar with that repo, so I have mostly questions rather than remarks.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -16,5 +16,3 @@ repos:
       - id: check-case-conflict
       - id: detect-private-key
       - id: check-yaml
-      - id: end-of-file-fixer
-      - id: trailing-whitespace
```
[the remark this answers is not in the document]
Comment (goern) [reply .pre-commit-config.yaml:20]: I think is just reordering and dedup?!

[the remark this answers is not in the document]
Comment (VannTen) [reply .pre-commit-config.yaml:20]: Wow, I'm distracted. Sorry for the noise.

**Review by VannTen — APPROVED** [review]
/lgtm

**Review by harshad16 — APPROVED** [review]
/lgtm
/approve
thanks :100:

**Pull request merged**